docs/install.md

```diff
@@ -0,0 +1,180 @@
+try adding the following libraries, then run pip3 again.
+``` shell
+    $ pip3 install --user urllib3 pyopenssl
+```
+
+## Windows 10
+
+Start by installng Python 3 using the latest Windows installer available from 
+https://www.python.org/downloads/windows/ and ensure that python.exe is in 
+the path.  
+
+Install `pyvcloud` and add to the PATH (needed for local installs):
+``` shell
+    C:\Users\Administrator>pip3 install --user pyvcloud
+    C:\Users\Administrator>set PATH=%PATH%;C:\Users\Administrator\AppData\Roaming\Python\Python36\Scripts
+```
+
+Administrator is just an example.  Other accounts work as well. 
+
+## Verify Installation
+
+Run the system-info.py script in the examples directory:  
+``` shell
+    $ cd examples
+    $ ./system-info.py 10.150.202.222 Org1 user1 mypassword
+    Logging in: host=10.150.202.222, org=Org1, user=user1
+    Fetching vCD installation info...
+    Key: name Value: VMware vCloud Director
+    Key: href Value: https://10.150.202.222/api/admin/
+    Key: type Value: application/vnd.vmware.admin.vcloud+xml
+    Logging out
+```
+The host, org, user, and password values should be specific to your
+vCD installation.
+
+## Installation with virtualenv
+
+It is also possible to install `pyvcloud` in a [virtualenv](https://docs.python.org/3/library/venv.html).  Quick commands to do so:
+``` shell
+    $ python3 -m venv $HOME/my_venv
+    $ . $HOME/my_venv
+    (my_venv) $ pip3 install pyvcloud
+```
+To terminate the virtual environment use the `deactivate` command. In
+this case we don't use the --user option as venv supplies its own location
+for packages.  Adding the --user option may cause odd installation behavior on
+some platforms.
+
+## Upgrade
+
+To upgrade an existing `pyvcloud` install, run:
+
+``` shell
+    $ pip3 install --user pyvcloud --upgrade
+```
+
+## Pre-releases
+
+The commands described above install the current stable version of `pyvcloud`. 
+To install a pre-release version, enter:
+
+``` shell
+    $ pip3 install --user pyvcloud --pre
+```
+
+And to upgrade a pre-release:
+
+``` shell
+    $ pip3 install --user pyvcloud --pre --upgrade
+```
+
+Installation from the current development version in GitHub:
+
+``` shell
+    $ pip3 install --user git+https://github.com/vmware/pyvcloud.git
+```
+
+Install specific Github or published versions:
+
+``` shell
+    $ pip3 install --user git+https://github.com/vmware/pyvcloud.git@19.1.0
+    (or)
+    $ pip3 install --user pyvcloud==19.1.0
+```
+
+## Installation from Local Source Files
+
+This is the standard installation for development.  Clone the code and
+install from source.
+``` shell
+    $ git clone https://github.com/vmware/pyvcloud.git
+    $ cd pyvcloud
+    $ python3 setup.py develop
+```
+Scripts will automatically pick up current pyvcloud source files including
+any changes you have made.  Use of virtualenv is recommended to avoid
+polluting your production Python installation.
```

Comment:
"Use of virtualenv is recommended to avoid polluting your production Python installation"
Better to highlight this somewhere at the top to prevent users from installing first and then reading this at the end of the page.

Reply:
Hi Sahithi, I would prefer to leave this as is.  The --user option is used consistently throughout, which means that if people follow the examples the they will not have problems with messing up their regular installation.  Also, I think we need to assume that users have some basic python knowledge or the docs will get rather cluttered.

Reply:
Okay.